src/requirements/data/majors/math.ts

```diff
@@ -0,0 +1,396 @@
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          ['MATH 3610', 'CS 2110', 'PHYS 1116', 'PHYS 2208', 'PHYS 2213', 'PHYS 2217']
+        ),
+        perSlotMinCount: [4, 1],
+        slotNames: ['3000+ Math Courses', 'Mathematical Models Course'],
+      },
+      'Computer Science': {
+        description:
+          '5 courses from (i) and (ii): ' +
+          '(i) At least one MATH course numbered 3000 or above, ' +
+          '(ii) 3 CS courses with significant mathematical content',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'CS 3220',
+            'CS 4110',
+            'CS 4160',
+            'CS 4210',
+            'CS 4220',
+            'CS 4620',
+            'CS 4670',
+            'CS 4700',
+            'CS 4740',
+            'CS 4744',
+            'CS 4775',
+            'CS 4780',
+            'CS 4786',
+            'CS 4787',
+            'CS 4810',
+            'CS 4812',
+            'CS 4814',
+            'CS 4820',
+            'CS 4830',
+            'CS 4850',
+            'CS 4852',
+            'CS 4860',
+          ],
+          [
+            'CS 3220',
+            'CS 4110',
+            'CS 4160',
+            'CS 4210',
+            'CS 4220',
+            'CS 4620',
+            'CS 4670',
+            'CS 4700',
+            'CS 4740',
+            'CS 4744',
+            'CS 4775',
+            'CS 4780',
+            'CS 4786',
+            'CS 4787',
+            'CS 4810',
+            'CS 4812',
+            'CS 4814',
+            'CS 4820',
+            'CS 4830',
+            'CS 4850',
+            'CS 4852',
+            'CS 4860',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
+        ),
+        perSlotMinCount: [1, 3, 1],
+        slotNames: ['3000+ Math Course', 'Mathematical CS Courses', 'MATH or CS Course'],
+      },
+      Economics: {
+        description:
+          '5 courses from (i), (ii), and (iii): ' +
+          '(i) At least one MATH course numbered 3000 or above, ' +
+          '(ii) At least three ECON courses with significant mathematical content, ' +
+          '(iii) A course in ORIE with significant mathematical content dealing with material of interest in economics',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'ECON 3130',
+            'ECON 3140',
+            'ECON 3810',
+            'ECON 3810',
+            'ECON 3825',
+            'ECON 4020',
+            'ECON 4110',
+            'ECON 4907',
+            'ECON 6090',
+            'ECON 6100',
+            'ECON 6130',
+            'ECON 6140',
+          ],
```

Comment:
Seems to be missing a couple courses like ECON 6190, ECON 6200, etc.
![Screen Shot 2021-04-17 at 10 10 16 AM](https://user-images.githubusercontent.com/54298311/115115948-1c492100-9f65-11eb-9984-c94551e82903.png)